src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
+  font-family: Roboto, Arial, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 0 50px;
+  max-width: 1200px;
+}
+
+.go-home {
+  margin: 10px 0;
+  height: 40px;
```

Comment:
no need to set `height` for this because you doing this for class `logo`

Reply:
and you can remove `go-home` class from your link and set `margin` for logo as well

Reply:
When I remove this height or this class, the logo remains 40px, but the link increases to 64px, the header increases as well, but there is a condition about 60px.

Reply:
> no need to set `height` for this because you doing this for class `logo`

When I remove this height or this class, the logo remains 40px, but the link increases to 64px, the header increases as well, but there is a condition about 60px.

Reply:
so in such case you can remove height from `logo` class, because there is no sense to set it twice